### PR TITLE
Add basic introduction to README

### PR DESCRIPTION
**Pull request by kytrinyx**
I'm not sure how much we need to put here. There will be more useful information on the mentors.exercism.io site as soon as we launch.

### CODE REVIEW

**Comment by iHiD** [comment]
@kytrinyx mentors.exercism.io actually ended up being mentoring.exercism.io due to tiredness :)

I'm ok with that if you are? I think I actually prefer it.

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# mentors
\ No newline at end of file
+# Exercism Mentors
+
+We are always looking for more mentors to help new people ramp up in the programming languages that Exercism supports.
+
+Read more about mentorship at Exercism on [mentors.exercism.io](https://mentors.exercism.io/).
```
Comment (iHiD) [inline README.md]: Currently mentoring.exercism.io

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+## How many mentors do we need?
+
+Well, that depends. For any given track, the minimum number of active mentors is *five*. This ensures that the burden is never too great for a single mentor, even when life gets busy for a bit, or you go on vacation. Beyond this, we need an extra mentor for every ten-twelve additional submissions per week.
+
+For reference, the most active track at the moment is the Python track, with about 150 submissions per day. This equates roughly to a need for about 90 active mentors.
```
Comment (iHiD) [inline README.md]: I think we should say "This equates roughly to a need for about 90 active mentors, each giving about 1 hour per week" or similar. This currently feels like we need 90 mentors on every night, which sounds intimidating.

File: README.md (hunk 1 above)
Comment (kytrinyx) [reply README.md]: That works for me!

File: README.md (hunk 2 above)
Comment (kytrinyx) [reply README.md]: Oooh, right! Yeah that's much better.

**Pull request merged**